src/v0/destinations/google_adwords_enhanced_conversions/data/trackConfig.json

```
@@ -129,5 +129,13 @@
     "metadata": {
       "type": "hashToSha256"
     }
+  },
+  {
+    "destKey": "jobId",
+    "sourceKeys": ["properties.jobId"],
```

```suggestion
    "sourceKeys": ["properties.jobId","messageId"],
```
let's add messageId as well as it can be used as a differentiator of events.

---

as we discussed jobId is supposed to be an integer. So we cannot do this . Ref: https://developers.google.com/google-ads/api/rest/reference/rest/v15/customers/uploadConversionAdjustments#ConversionAdjustment:~:text=%3A%20boolean%2C-,%22jobId%22%3A%20integer,-%7D